### PR TITLE
NMS-16028: Added doc entry and additional test

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-16028

### CODE REVIEW

File: docs/modules/operation/pages/quick-start/inventory.adoc [hunk 1]
```diff
@@ -49,6 +49,8 @@ image::provisioning/SNMP_Config.png["SNMP community name configuration page with
 These settings are automatically optimized into `snmp-config.xml`.
 For an example of the resulting XML configuration, see <<deep-dive/provisioning/xml-samples.adoc#SNMP-community-xml, Configure SNMP community names>> in the Deep Dive section.
 
+NOTE: It is possible to use meta-data expressions in the fields Security Name, Read Community, Write Community, Auth Passphrase and Privacy Passphrase to reference credentials stored in the Secure Credentials Vault.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/quick-start/inventory.adoc:52]:
```suggestion
NOTE: It is possible to use metadata expressions in the fields Security Name, Read Community, Write Community, Auth Passphrase, and Privacy Passphrase to reference credentials stored in the Secure Credentials Vault.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor changes in the text. thank you!

**Review by Bonrob2 — APPROVED** [review]
docs LGTM! Thank you!

File: docs/modules/operation/pages/quick-start/inventory.adoc [hunk 2]
```diff
@@ -49,6 +49,8 @@ image::provisioning/SNMP_Config.png["SNMP community name configuration page with
 These settings are automatically optimized into `snmp-config.xml`.
 For an example of the resulting XML configuration, see <<deep-dive/provisioning/xml-samples.adoc#SNMP-community-xml, Configure SNMP community names>> in the Deep Dive section.
 
+NOTE: It is possible to use metadata expressions in the fields Security Name, Read Community, Write Community, Auth Passphrase, and Privacy Passphrase to reference credentials stored in the Secure Credentials Vault.
+
```
Comment (cgorantla) [inline docs/modules/operation/pages/quick-start/inventory.adoc:53]: we should also mention this only works with global metadata not with any node, interface etc.

Comment (christianpape) [reply docs/modules/operation/pages/quick-start/inventory.adoc:54]: done.

**Review by Bonrob2 — APPROVED** [review]
changed LGTM.

**Review by cgorantla — APPROVED** [review]

**Pull request merged**